### ISSUE TITLE
Question About the Frozen Speaker Encoder Configuration

### ISSUE THREAD

**Issue by DommyWorld** [TaurenMountain/PS4#3]
Hello,

I tried to reproduce the training code, but I found some inconsistencies that I would like to clarify.

In the comments of the code, it is mentioned that for the `FrozenSpeakerEncoder`, the English dataset should use `voxceleb_resnet34_LM`, while the Chinese dataset should use `cnceleb_resnet34_LM`. The paper also states that a “frozen ResNet34” is used.

However, based on the actual implementation, the configuration parsing logic does not seem to support either of these two models. Instead, it appears to support `voxceleb_ECAPA512_LM`. Moreover, the `get_speaker_model` method does not seem to support `ResNet34` as a model type either.

Could you please clarify what is going on here? Is the code outdated and has not been updated to match the paper, or should we actually use the `voxceleb_ECAPA512_LM` model for reproduction?

Thank you for your help!

**Comment by ningwanyi** [comment]
Thanks for catching this. Actually, the method involves both of the `ECAPA_TDNN_GLOB_c512` and `ResNet34` (`voxceleb_resnet34_LM` / `cnceleb_resnet34_LM`). 

- `ECAPA_TDNN_GLOB_c512` is frozen and inherited from the `bsrnn_ecapa_vox1` pretrained checkpoint, which is *inside* the BSRNN separator. You can see this `get_speaker_model` call on line `185` of file `wesep_ps4/wesep_real_tse/wesep/models/bsrnn_legacy.py`:

  ```
  self.spk_model = get_speaker_model(spk_model)(**spk_args)
  ```

-  `ResNet34` is frozen and used only for the L_SIM similarity ranking loss, matching the challenge evaluation side. You can see this `get_speaker_model` call on line `798-803` of file `train.py`:

  ```
  spk_model_name   = spk_cfg.get("model", "ResNet34")
  spk_model_kwargs = spk_cfg.get("model_args", {})

  from wespeaker.models.speaker_model import get_speaker_model
  spk_cls = get_speaker_model(spk_model_name)
  self.encoder = spk_cls(**spk_model_kwargs)
  ```

**Root cause:** `resnet.py` was not packaged into `wesep_ps4/wespeaker/wespeaker/models/`, and `speaker_model.py` only kept the `ECAPA_TDNN` dispatch branch. So `get_speaker_model("ResNet34")` always failed.

**Fix:** added `resnet.py` (copied from upstream wespeaker) and restored the `ResNet` branch in `get_speaker_model()`.
